db/migrate/20150818181427_update_tenant_divisible_on_existing_rows.rb

```diff
@@ -1,7 +1,8 @@
 class UpdateTenantDivisibleOnExistingRows < ActiveRecord::Migration
   class Tenant < ActiveRecord::Base
   end
+
   def up
-    Tenant.where(:divisible => nil).each { |t| t.update_attribute(:divisible, true) }
+    Tenant.where(:divisible => nil).update_all(:divisible => true)
```

Comment:
All data migrations should have specs.  Additionally, it should also be wrapped in a `say_with_time` block that describes what is changing.  See other data migrations for examples.